### DISCUSSION TITLE
Welcome to sp500 Discussions!

### DISCUSSION THREAD

**Discussion by fja05680** [fja05680/sp500#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rkkc2003** [comment]
Sorry, I've posted in the wrong area.
